packages/teams-js/test/public/video.spec.ts

```diff
@@ -90,17 +90,6 @@ describe('video', () => {
           expect(message?.args?.length).toBe(1);
           expect(message?.args).toEqual([registerForVideoFrameParameters.config]);
         });
-
-        it('should not send default message when register video frame handler', async () => {
```

Comment:
why don't you need this unit test anymore?

Reply:
recovered, I thought it was useless.